plugins/session_notifier.rb

```diff
@@ -315,6 +333,30 @@ def send_text_to_gotify(session)
         end
       end
 
+      def send_text_to_serverjang(session)
+        # https://sct.ftqq.com/sendkey
+        platform = "#{session.platform}"
+        arch = "#{session.arch}"
+        tunnel = "#{session.tunnel_to_s}"
+        uri_parser = URI.parse(serverjang_webhook)
+        params = {}
+        params["title"] = "You have new #{session.type} session"
+        params["desp"] = 'OS:%s, tunnel:%s, Arch:%s' %[platform,tunnel,arch]
```

Comment:
There is no reason to do this when we have string interpolation support. This would also remove the need to define 3 additional variables purely for the purpose of string interpolation.

Suggestion:
```suggestion
        uri_parser = URI.parse(serverjang_webhook)
        params = {}
        params["title"] = "You have new #{session.type} session"
        params["desp"] = "OS: #{platform}, tunnel: #{tunnel}, Arch: #{arch}"
```

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/16192/commits/5a8cf3ea95aea77f220a8548c89535868fbc0c34